docs/dogfood/index.md

```diff
@@ -0,0 +1,143 @@
+# Dogfooding
+
+This guide explains how to [dogfood](https://www.techopedia.com/definition/30784/dogfooding) coder for employees at Coder.
+
+## How to 
+
+The following explains how to do certain things related to dogfooding.
+
+### Dogfood using Coder's Deployment
+
+1. Go to [https://dev.coder.com](https://dev.coder.com/) 
+    - If you see a dialog/pop-up, hit "Cancel"
```

Comment:
We should add context that this is because of Rippling ;p. Otherwise it could be confusing.